base/kbd/stone.yml

```diff
@@ -0,0 +1,24 @@
+#
+# SPDX-FileCopyrightText: © 2020-2022 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : kbd
+version     : 2.5.1
+release     : 1
+homepage    : https://kbd-project.org/
+upstreams   :
+    - https://mirrors.edge.kernel.org/pub/linux/utils/kbd/kbd-2.5.1.tar.xz : ccdf452387a6380973d2927363e9cbb939fa2068915a6f937ff9d24522024683
+summary     : Linux keyboard tools
+description : |
+    The kbd project contains utilities for managing Linux console (Linux console, virtual terminals, keyboard, etc.)
+license     : GPL-2.0-or-later
+builddeps   :
+    - pkgconfig(pam)
+    - binary(gzip)
```

Comment:
FWIW I've been moving `binary` deps above `pkgconfig` ones.

Reply:
Yeah, forgot to sort it